specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md

```diff
@@ -30,12 +30,31 @@ description: HDInsight Containers Management Client
 openapi-type: arm
 openapi-subtype: rpaas
 azure-arm: true
-tag: package-2023-06-preview
+tag: package-preview-2023-11
 ```
 
 ### Suppression
 
 
+### Tag: package-preview-2023-11
+
+These settings apply only when `--tag=package-preview-2023-11` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-11'
+input-file:
+  - preview/2023-11-01-preview/hdinsight.json
+  
+suppressions:
+  - code: ResourceNameRestriction
+    reason: Keep compatibility with old API version.
+  - code: TrackedResourcePatchOperation
+    reason: This is a false positive, the "tags" property is defined in TrackedResource.
+  - code: OperationIdNounVerb
+    reason: The operation id is valid.
+  - code: EnumInsteadOfBoolean
+    reason: The boolean property is expected.
+```
```

Comment:
Please supress errors for specific paths and elements rather than blanket suppressions for the entire file : 

[https://eng.ms/docs/products/azure-developer-experience/design/specs-pr-guides/pr-suppressions](https://eng.ms/docs/products/azure-developer-experience/design/specs-pr-guides/pr-suppressions)